worker/src/RTC/SeqManager.cpp

```diff
@@ -77,28 +77,30 @@ namespace RTC
 		if (!this->dropped.empty())
 		{
 			// Delete dropped inputs older than input - MaxValue/2.
-			size_t droppedSize = this->dropped.size();
-			auto it            = this->dropped.lower_bound(input - MaxValue / 2);
+			size_t droppedCount = this->dropped.size();
+			auto it        = this->dropped.lower_bound(input - MaxValue / 2);
```

Comment:
Please run `make lint` in `worker/` directory,. This will fail. Auto-fix it by running `make format`.